students/njschafi/Lesson09/test_mailroom9.py

```diff
@@ -0,0 +1,172 @@
+"""test code for lesson 9 mailroom"""
+# NEIMA SCHAFI, LESSON 9 Assignment test code
+
+import pytest
+import os
+from fun import Donors, Donor
```

Comment:
You are correct that it should read "mailroom9" instead of "fun". I forgot to update the test code when I renamed the "mailroom9" file. Thanks for the feedback!